src/strife/f_wipe.c

```diff
@@ -88,6 +95,15 @@ wipe_initColorXForm
 // * Rogue modified the unused ColorXForm wipe in-place in order to implement 
 //   their distinctive crossfade wipe.
 //
+#ifdef CRISPY_TRUECOLOR
+static const uint8_t alpha_table[] = {
```

Comment:
Isn't this just steps of 8? Then we would just need to pass `fade_counter * 8` to `I_BlendOver()`.

Reply:
Yes, exactly. Have a habit to tablify whatever is possible, though previous one approach should work.

Reply:
I think it's easier to read `8 * x` instead of `multiples_of_eight[x]`. I prefer few, focussed, unambigious code changes.